ibc-chain-registry/src/paths.rs

```diff
@@ -129,26 +115,26 @@ mod tests {
         use std::str::FromStr;
 
         let path = r#"{
-            "$schema": "http://json-schema.org/draft-07/schema#",
-            "chain-1": {
-                "chain-name": "chain-1",
-                "client-id": "tendermint-1",
-                "connection-id": "connection-1"
+            "$schema": "http://json_schema.org/draft_07/schema#",
```

Comment:
I think we should keep the dashes here:

```suggestion
            "$schema": "http://json-schema.org/draft-07/schema#",
```